### ISSUE TITLE
A11y_Visual Studio Code Client_Extension_Rating_Screen Reader: Jaws is not announcing the tooltip information available on "rating" link.

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/vscode#203636]
[([Check out Accessibility Insights!](https://accessibilityinsights.io/))]- Identify accessibility bugs before check-in and make bug fixing faster and easier.

### GitHubTags:
#A11yTCS; #A11ySev3; #DesktopApp; #Win32; #Visual Studio Code Client; #BM-VisualStudioCodeClient-Win32-Jan2024; #WCAG1.3.1; #ScreenReader; #JAWS; #NVDA; #A11yMAS;

### Environment and OS details:
Visual Studio Code insider version 1.86.0
Microsoft Windows 11 Enterprise 22H2 Build 22621.3007
JAWS
NVDA Version: 2022.2.3

### Repro Steps:
1. Open Visual Studio Code Insiders editor
2. Using "Ctrl+Shift+X" key navigate to the extension side view bar option.
3. Select an extension and using "Tab" and "Arrow" keys navigate through the extension details window on the RHS.
4. Tab till rating link.
5. Verify whether the Jaws is announcing the tooltip information available on "rating" link or not.

### Actual Result:
Jaws is not announcing the tooltip information available on "rating" link when we are navigating using the tab key.

**Note: Issue is not repro'ing using the NVDA**

### Expected Result:
Jaws should announce the tooltip information available on "rating" link.

### User Impact:
Screen reader only users will not be able to get any information about the rating.

### Attachment:
**JAWS**:

https://github.com/microsoft/vscode/assets/124244478/ce118b95-6edd-4c16-a903-432a42a5697d

**NVDA: Issue is not repro'ing using the NVDA.**

![Bug_203636_NVDA](https://github.com/microsoft/vscode/assets/124244478/5a9f12ef-9bd0-46c1-92f5-66d833fd9346)

**Comment by meganrogge** [comment]
Since the issue is not reproducing when using NVDA, closing this as not actionable on our end. @isidorn pls reopen if you disagree
